src/smart-components/access-management/UserGroupsTable.tsx

```diff
@@ -86,7 +122,7 @@ const UserGroupsTable: React.FunctionComponent = () => {
     group.roleCount,
     group.workspaces || '?', // not currently in API
     formatDistanceToNow(new Date(group.modified), { addSuffix: true }),
-    {
+    enableActions && {
       cell: <ActionsColumn items={ROW_ACTIONS} />,
       props: { isActionCell: true },
     },
```

Comment:
@CodyWMitchell you can also pass ID to the rows like so
```
const rows = groups.map((group: any) => ({
    id: group.uuid,
    row: [
      group.name,
      group.description ? (
        <Tooltip isContentLeftAligned content={group.description}>
          <span>{group.description.length > 23 ? group.description.slice(0, 20) + '...' : group.description}</span>
        </Tooltip>
      ) : (
        <div className="pf-v5-u-color-400">No description</div>
      ),
      group.principalCount,
      group.serviceAccounts || '?', // not currently in API
      group.roleCount,
      group.workspaces || '?', // not currently in API
      formatDistanceToNow(new Date(group.modified), { addSuffix: true }),
      enableActions && {
        cell: <ActionsColumn items={ROW_ACTIONS} />,
        props: { isActionCell: true },
      },
    ],
  }));
```
this way you can get rid of the useEffect, you can use the ID in `matchOption`, and the add modal can receive just a list of IDs that will be sent to the API